components/homepage/Services.tsx

```diff
@@ -12,10 +12,34 @@ const Services = (props: Props) => {
           Provide
         </h2>
         <div className="flex flex-col justify-center gap-[2.5rem] md:flex-row md:flex-wrap">
-          <ServiceCard />
-          <ServiceCard />
-          <ServiceCard />
-          <ServiceCard />
+          <ServiceCard
+            title={"Analytics & SEO"}
+            body={
+              "Implement tracking & analytics tools to monitor website performance & improve SEO."
+            }
+            image={"/seo.svg"}
+          />
+          <ServiceCard
+            title={"Front-End Dev"}
+            body={
+              "Front-end dev includes creating the user interface and interactive elements of websites and apps."
+            }
+            image={"/Pointer.svg"}
+          />
+          <ServiceCard
+            title={"Back-End Dev"}
+            body={
+              "Front-end includes creating the user interface and interactive elements of websites and apps."
+            }
+            image={"/Code.svg"}
+          />
+          <ServiceCard
+            title={"Web Optimization"}
+            body={
+              "Web optimization focuses on improving website performance for a smoother user experience"
+            }
+            image={"/Level.svg"}
+          />
```

Comment:
Same idea here. If we're looping over data that you may want to change in the future- consider pulling it from elsewhere like sanity. This component can loop over that data instead of hardcoding these components.